config/packages/nelmio_security.yaml

```diff
@@ -49,3 +49,8 @@ nelmio_security:
                 - 'self'
             block-all-mixed-content: true # defaults to false, blocks HTTP content over HTTPS transport
             # upgrade-insecure-requests: true # defaults to false, upgrades HTTP requests to HTTPS transport
+
+    forced_ssl:
+       hsts_max_age: 2592000
+       hsts_subdomains: true
+       redirect_status_code: 301
```

Comment:
Tests are RED because they don't run in a HTTPS-like environment.

It should be possible to change our test to make them believe they hit Symfony with ssl:

```
static::createClient(array(),array('HTTPS' => true));
```

Could you try?